.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+## Overview
+
+<!-- Please insert what the number of the issue you are addressing. (If an issue doesn't exist for this PR to address, please open one at https://github.com/desktop/desktop/issues before opening this PR.) -->
```

Comment:
```suggestion
<!-- 
What issue are you addressing? e.g. #1234 
If there is no issue for this pull request (PR), please create one before opening your PR.
Issues can be created at https://github.com/desktop/desktop/issues) 
-->
```

I think ☝️ 
* Increases readability by separating content from markup
* Question makes clear what we're looking for
* Helps guide user by giving an example
* Defines acronym before using it